src/frontend/context/ManuellJournalførContext.tsx

```diff
@@ -0,0 +1,634 @@
+    };
+
+    const finnValgtDokument = (ressurs: Ressurs<IDataForManuellJournalføring>) => {
+        return finnDokument(ressurs, valgtDokumentId);
+    };
+
+    const settValgtDokumentInfo = (dokumentInfo: IDokumentInfo) => {
+        oppdatertData.status === RessursStatus.SUKSESS &&
+            settOppdatertData({
+                ...oppdatertData,
+                data: {
+                    ...oppdatertData.data,
+                    journalpost: {
+                        ...oppdatertData.data.journalpost,
+                        dokumenter: oppdatertData.data.journalpost.dokumenter?.map(dokument => {
+                            return dokument.dokumentInfoId === valgtDokumentId
+                                ? dokumentInfo
+                                : dokument;
+                        }),
+                    },
+                },
+            });
+    };
+
+    const settDokumentTittel = (dokumentTittel: string) => {
+        const valgt = finnValgtDokument(oppdatertData);
+        if (!valgt) {
+            return;
+        }
+        valgt.tittel = dokumentTittel;
+        valgt.brevkode = hentBrevkode(valgt.tittel);
+        settValgtDokumentInfo(valgt);
+    };
+
+    const settLogiskeVedlegg = (logiskeVedleggNavn: Array<string>) => {
+        const valgt = finnValgtDokument(oppdatertData);
+        if (!valgt) {
+            return;
+        }
+
+        valgt.logiskeVedlegg = logiskeVedleggNavn.map(vedlegg => {
+            return {
+                tittel: vedlegg,
+                logiskVedleggId: '0', // this id is not nullable but ignored by backend so set it to whatever string
+            };
+        });
+        settValgtDokumentInfo(valgt);
+    };
+
+    const hentBrevkode = (dokumentTittel: string | undefined): string => {
+        return BrevkodeMap.get(dokumentTittel) || '';
+    };
+
+    const settFagsak = (fagsak: Ressurs<IFagsak | undefined>) => {
+        if (
+            oppdatertData.status === RessursStatus.SUKSESS &&
+            fagsak.status === RessursStatus.SUKSESS
+        ) {
+            settOppdatertData({
+                ...oppdatertData,
+                data: {
+                    ...oppdatertData.data,
+                    fagsak: fagsak.data,
+                },
+            });
+        }
+    };
+
+    const hentFagsak = async (personId: string) => {
+        return axiosRequest<IFagsak | undefined, Ressurs<IFagsak | undefined>>({
+            method: 'GET',
+            url: `/familie-ba-sak/api/fagsakForPerson`,
+            headers: {
+                personIdent: personId,
+            },
+        }).then((fagsak: Ressurs<IFagsak | undefined>) => {
+            return fagsak;
+        });
+    };
+
+    const endreBruker = async (personId: string) => {
+        const hentetPerson = await axiosRequest<IPersonInfo, void>({
+            method: 'GET',
+            url: '/familie-ba-sak/api/person',
+            headers: {
+                personIdent: personId,
```

Comment:
Same her with ident in the body (if this is an old endpoint we can take this in a separate PR)

Reply:
It is an old endpoint